velox/dwio/dwrf/common/Config.cpp

```diff
@@ -182,7 +182,7 @@ Config::Entry<const std::vector<std::vector<std::string>>>
 
 Config::Entry<uint32_t> Config::MAP_FLAT_MAX_KEYS(
     "orc.map.flat.max.keys",
-    20000);
+    30000);
```

Comment:
@sdruzkin, I recall there's a discussion on bumping to 25k instead. Could you remind me of the resolution there?

I'm also curious how much memory increase a 5K/10K per column bump tends to bring, in case you already have a measurement (from vader maybe?).